datagateway_api/src/datagateway_api/database/models.py

```diff
@@ -350,6 +374,210 @@ class DATACOLLECTIONPARAMETER(Base, EntityHelper, metaclass=EntityMeta):
+    __pluralfieldname__ = "dataPublicationTypes"
+    __table_args__ = (Index("UNQ_DATAPUBLICATIONTYPE_0", "FACILITY_ID", "NAME"),)
+
+    id = Column("ID", BigInteger, primary_key=True)
+    createId = Column("CREATE_ID", String(255), nullable=False)
+    createTime = Column("CREATE_TIME", DateTime, nullable=False)
+    modId = Column("MOD_ID", String(255), nullable=False)
+    modTime = Column("MOD_TIME", DateTime, nullable=False)
+    name = Column("NAME", String(255), nullable=False)
+    description = Column("DESCRIPTION", String(255), nullable=False)
+    datapublicationId = Column(
+        "DATAPUBLICATION_ID", ForeignKey("DATAPUBLICATION.ID"), nullable=False,
+    )
+
+    facilityId = Column("FACILITY_ID", ForeignKey("FACILITY.ID"), nullable=False)
+
+    DATAPUBLICATION = relationship(
+        "DATAPUBLICATION",
+        primaryjoin="DATAPUBLICATIONTYPE.datapublicationId == DATAPUBLICATION.id",
+        backref="dataPublicationTypes",
+    )
+
+    FACILITY = relationship(
+        "FACILITY",
+        primaryjoin="DATAPUBLICATIONTYPE.facilityId == FACILITY.id",
+        backref="dataPublicationTypes",
+    )
+
+
+class DATAPUBLICATIONUSER(Base, EntityHelper, metaclass=EntityMeta):
+    __tablename__ = "DATAPUBLICATIONUSER"
+    __singularfieldname__ = "dataPublicationUser"
+    __pluralfieldname__ = "dataPublicationUsers"
+    __table_args__ = (
+        Index(
+            "UNQ_DATAPUBLICATIONUSER_0", "PUBLICATION_ID", "USER_ID", "CONTRIBUTORTYPE",
+        ),
+    )
+
+    id = Column("ID", BigInteger, primary_key=True)
+    createId = Column("CREATE_ID", String(255), nullable=False)
+    createTime = Column("CREATE_TIME", DateTime, nullable=False)
+    modId = Column("MOD_ID", String(255), nullable=False)
+    modTime = Column("MOD_TIME", DateTime, nullable=False)
+    orderKey = Column("ORDERKEY", String(255), nullable=False)
+    givenName = Column("GIVENNAME", String(255), nullable=False)
+    fullName = Column("FULLNAME", String(255), nullable=False)
+    familyName = Column("FAMILYNAME", String(255), nullable=False)
+    contributorType = Column("CONTRIBUTORTYPE", String(255), nullable=False)
+    publicationId = Column(
+        "PUBLICATION_ID", ForeignKey("DATAPUBLICATION.ID"), nullable=False,
+    )
+
+    userID = Column("USER_ID", ForeignKey("USER_.ID"), nullable=False)
+    affiliationId = Column(
+        "AFFILIATION_ID", ForeignKey("AFFILIATION.ID"), nullable=False,
+    )
+
+    USER = relationship(
+        "USER",
+        primaryjoin="DATAPUBLICATIONUSER.userID == USER.id",
+        backref="dataPublicationUsers",
+    )
+
+    PUBLICATION = relationship(
+        "DATAPUBLICATION",
+        primaryjoin="DATAPUBLICATIONUSER.publicationId == DATAPUBLICATION.id",
+        backref="dataPublicationUsers",
+    )
+
+    AFFILIATION = relationship(
+        "AFFILIATION",
+        primaryjoin="DATAPUBLICATIONUSER.affiliationId == AFFILIATION.id",
+        backref="dataPublicationUsers",
```

Comment:
The correct backref is `user`, please can this be corrected?

Reply:
Yes, changed